buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
 
     // This is required until Gradle TestSuites are used, note no version is specified as this syncs
     // with JUnit from gradle.
-    testRuntimeOnly(group: 'org.junit.platform', name: 'junit-platform-launcher')
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
```

Comment:
The assignment here is not problematic, but we should migrate away from it because we should at some point move to Kotlin as this is the recommended syntax and the syntax above is gradle specific :(